spec/models/video_spec.rb

```diff
@@ -2,24 +2,27 @@
 
 describe Video do
   it {should belong_to (:category)}
+  it {should have_many(:reviews).order("created_at DESC")}
   it {should validate_presence_of (:title)}
   it {should validate_presence_of (:description)}
 
   describe "#search_by_title" do
-    context "when there's no match" do
+    let(:subject) {Video.search_by_title("some title")}
+
+    context "no match" do
       it "returns an empty array" do
-        expect(Video.search_by_title("some title")).to eq([])
+        expect(subject).to eq([])
```

Comment:
I don't think this is a good refactor, its less evident what subject is. Also subject is a bit different in RSpec than defining it with `let`. I would avoid using it for now.